buildSrc/build.gradle

```diff
@@ -177,6 +177,12 @@ if (project != rootProject) {
 
     integTestRuntimeOnly(project(":libs:elasticsearch-core"))
   }
+  
+  eclipse {
+	classpath {
+		plusConfigurations += [configurations.integTestRuntimeClasspath]
+	}
```

Comment:
Probably better to switch to spaces here so it renders well everywhere.

Reply:
fixed